### PR TITLE
NAMU page is done

### PR DESCRIPTION
**Pull request by mixelio**
[DEMO LINK](https://mixelio.github.io/Museum_2/).

### CODE REVIEW

**Review by anastasiia-tilikina — CHANGES_REQUESTED** [review]
You did really great job! your landing looks pretty much well)
But let's fix some issues though 

1. The pic should be bigger on desktop (check design)
image

2. add some hover effect on this button
image
and this
image

3. when I use autocomplete my input text color is invisible)
image

4. On some screen width the image overlaps the button
image

5. On mobile your button should be 100% width of container.
image

as well as these buttons
image

Basically all buttons) check design)

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
1. Need to remove background color after autocomplete styles
image

**Review by maxim2310 — APPROVED** [review]
Good work!!!
